.rubocop.yml

```diff
@@ -13,7 +13,25 @@ Metrics/BlockLength:
 ModuleLength:
   Enabled: false
 
-Style/EmptyMethod:
+Style/GlobalVars:
+  Enabled: false
+
+Style/IndentArray:
+  Enabled: false
+
+Style/AlignHash:
+  Enabled: false
+
+Style/Lambda:
+  Enabled: false
+
+Style/DoubleNegation:
```

Comment:
Looks easy to fix in code. If you really, really need double negation, just disable the rule locally using comments.